virtcontainers/kata_agent.go

```diff
@@ -70,6 +70,7 @@ var (
 	kataEphemeralDevType = "ephemeral"
 	ephemeralPath        = filepath.Join(kataGuestSandboxDir, kataEphemeralDevType)
 	grpcMaxDataSize      = int64(1024 * 1024)
+	maxHostnameLen       = 64
```

Comment:
Why add a new var?

Reply:
It was originally declared in the hyperstart_agent.go, but is needed here too, so I moved it :)

Reply:
I see.